packages/contract/lib/utils/index.js

```diff
@@ -133,22 +133,31 @@ const Utils = {
   },
 
   // Extracts optional tx params from a list of fn arguments
-  getTxParams(methodABI, args) {
+  getTxParams(methodABI, args, ignoreDefaultGasPriceParams = false) {
     const constructor = this;
 
     const expectedArgCount = methodABI ? methodABI.inputs.length : 0;
 
-    let tx_params = {};
+    let txParams = {};
```

Comment:
nice

Reply:
Haha yeah figured I'd change that while I was at it. :)